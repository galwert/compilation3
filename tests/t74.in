bool foo(bool x){
	while(1==2){
		byte y = 1b if(true) else 0b;
		return y==2 or 2<=y;
		//
		return (true if(true) else (1==2));
		bool hi = true if(true) else  1b==2 ;
		//if u have type missmatch it's because bison thinks of it like that:bool hi = (true if(true) else  1b)==2 ;
		bool bb = true if(true) else  1==2 ;
		break;
		continue;
	}
	return false if(5) else false;
}