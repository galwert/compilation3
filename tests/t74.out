---end scope---
y BYTE 0
hi BOOL 1
bb BOOL 2
---end scope---
line 13: type mismatch
